### PR TITLE
[FT:] fix pause tests

### PR DESCRIPTION
**Pull request by dmaish**
#### Problem
- the pause test misses test to check if the request is a 'POST' request.
-the test_it_works doesn't assert the expected status code and check's status.

#### Solution
- create a testcase to check if only 'POST' requests are allowed.
- create the assert statements to test the expected status code and check's status.

[https://www.pivotaltracker.com/story/show/158729308]

### CODE REVIEW

**Comment by miritih** [comment]
Test working as expected.

File: hc/api/tests/test_pause.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 from hc.api.models import Check
 from hc.test import BaseTestCase
```
Comment (jmutuku95) [inline hc/api/tests/test_pause.py]: To comply with pep8, consider making this import on the first line and remove the blank white space

File: hc/api/tests/test_pause.py [hunk 2]
```diff
@@ -13,15 +15,32 @@ def test_it_works(self):
...
         self.assertEqual(r.status_code, 400)
 
         ### Test that it only allows post requests
+
+    def test_only_allows_post_requests(self):
+        check = Check(user=self.bob, status="up")
+        check.save()
+
+        url = "/api/v1/checks/%s/pause" % check.code
+        r = self.client.get(url, "", content_type="application/json",
```
Comment (jmutuku95) [inline hc/api/tests/test_pause.py]: Use meaningful variable names. Instead of `r`, try using `response`

File: hc/api/tests/test_pause.py [hunk 3]
```diff
@@ -13,15 +15,32 @@ def test_it_works(self):
...
+    
         url = "/api/v1/checks/%s/pause" % check.code
-        r = self.client.post(url, "", content_type="application/json",
+        r = self.client.post(url,"", content_type="application/json",
                              HTTP_X_API_KEY="abc")
-
+        
         self.assertEqual(r.status_code, 400)
 
         ### Test that it only allows post requests
```
Comment (jmutuku95) [inline hc/api/tests/test_pause.py]: Remove prompts, add comments to explain code

**Comment by dmaish** [comment]
branch  was checked out from wrong branch.Not develop.

**Pull request closed without merging**